src/bin/cargo/commands/package.rs

```diff
@@ -5,6 +5,8 @@ use cargo::ops::{self, PackageOpts};
 pub fn cli() -> Command {
     subcommand("package")
         .about("Assemble the local package into a distributable tarball")
+        .arg_index("Registry index URL to prepare the package for")
+        .arg_registry("Registry to prepare the package for")
```

Comment:
Commit message and PR should talk of why these are needed

Reply:
(this will also help serve as the stabilization report for these flags when proposing to merge these)